helm_deploy/values-dev.yaml

```diff
@@ -22,5 +22,6 @@ ingress:
 env:
   JWT_PUBLIC_KEY: "LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0NCk1JSUJJakFOQmdrcWhraUc5dzBCQVFFRkFBT0NBUThBTUlJQkNnS0NBUUVBemN4N1lidzJCQVd2N1NzRUlVR1ANCml5SlNJRGd0cUJ4MTlUd203dUkzVFg1elJjclpsVXhjRW1QSkdVZ3krRDJKSWhWbHFtZXF3R1YyQ05PcVpCZ0cNCmo4ZUpHQTU5aUlUemU4ZG1SSk5JYzdsNmxESmc5RE5KVk9pTHFVbFpGRENJcXplSTYzb3E2dWhjY2c1RFBpVE4NCnFPR1pjOHVwTit3NWRack52KzJHTGd4SzJwZXRNVS9CaFllV2YzS3ZZSU8xNnYxdXZuRk9HTzEzSG9XdTVCdGQNClN0L1RnY2xGaFZMRWRHdzdYYmlZSG5OWklkaHdhTlFpWWdtWG1qWlZkTXlDUERNbzEwTGtWMXAzVXkxNXBNTXgNClVwc2xKYU8wNlZIYXJtY3ZWYzNleHg5NlpHTjE2T2U4ZWZoeG5Rdmhzd0ZrbXlYT25sSForNDI1MnJHcHlKTG8NCmx3SURBUUFCDQotLS0tLUVORCBQVUJMSUMgS0VZLS0tLS0="
   ELITE2API_ENDPOINT_URL: "https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api/"
+  OAUTH_ENDPOINT_URL: "https://gateway.t3.nomis-api.hmpps.dsd.io/auth"
```

Comment:
this is going to get confusing - does the URL end in a `/` or doesn't it?  Worth making consistent with elite2 url above - other apps all end in a slash (although I would prefer it if none of them did - I think I've had that conversation with @andrewrlee before)

Reply:
Looking at the elite url code it adds a double forward slash at the moment :( I should just fix it.